webrtc.html

```diff
@@ -4029,7 +4112,7 @@ <h4><dfn>RTCSessionDescription</dfn> Class</h4>
         session descriptions.</p>
         <div>
           <pre class="idl" data-tests="idlharness.https.window.js"
->[Exposed=Window, Constructor(RTCSessionDescriptionInit descriptionInitDict)]
+>[Exposed=Window, Constructor(optional RTCSessionDescriptionInit descriptionInitDict)]
```

Comment:
Editorial nit: Does "optional" make sense here? If it's missing, then implicitly type is missing, so we are throwing TypeError anyway. Would removing "optional" here have the same effect, only it would not lead the reader to believe it is optional until further reading?

Reply:
It MUST be optional according to [WebIDL](https://heycam.github.io/webidl/#dfn-optional-argument-default-value): *"If the type of an argument is a dictionary type ..., and that dictionary type and its ancestors have no required members, and the argument is either the final argument or is followed only by optional arguments, then the argument must be specified as optional ..."*